js/components/portal-dashboard/feedback/feedback-note-toggle.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { FeedbackLevel } from "../../../util/misc";
+
+import css from "../../../../css/portal-dashboard/feedback/feedback-note-toggle.less";
+
+interface IProps {
+  feedbackLevel: FeedbackLevel;
+}
+
+export const FeedbackNoteToggle: React.FC<IProps> = (props) => {
+
+  const handleFeedbackNoteButtonClick = () => {
+    // show feedback note modal
+  };
+
+  const noteLabel = props.feedbackLevel === "Activity" ? "activity-level" : "question-level";
+
+  return (
+    <div className={css.feedbackNoteToggle}>
+      <button className={css.feedbackNoteToggle__button} onClick={handleFeedbackNoteButtonClick}></button> <div>Note on {noteLabel} feedback</div>
```

Comment:
Watch line length.  This also might be easier to read with elements on their own lines and removal of the closing button tag:
```
      <button className={css.feedbackNoteToggle__button} onClick={handleFeedbackNoteButtonClick} />
      <div>Note on {noteLabel} feedback</div>
```